modules/setting/setting.go

```diff
@@ -1215,6 +1232,22 @@ func NewContext() {
 		// Explicitly disable credential helper, otherwise Git credentials might leak
 		git.GlobalCommandArgs = append(git.GlobalCommandArgs, "-c", "credential.helper=")
 	}
+
+	sec = Cfg.Section("indexer")
```

Comment:
Can we split this out to modules/setting/indexer.go?

Reply:
done.